example/src/main/java/io/netty/example/http/websocketx/server/WebSocketFrameHandler.java

```diff
@@ -21,15 +21,15 @@
 import io.netty.channel.SimpleChannelInboundHandler;
 import io.netty.handler.codec.http.websocketx.TextWebSocketFrame;
 import io.netty.handler.codec.http.websocketx.WebSocketFrame;
-import org.slf4j.Logger;
```

Comment:
Why is this change mixed in?

Reply:
Because this dependency was pulled in transitively which then failed after removing the dependencies for the example module ;). 

That said let me maybe just do a pr against 4.1 that removes the usage, pull it in and then rebase to make it more clear.

Reply:
@ejona86 https://github.com/netty/netty/pull/8582